menu.py

```diff
@@ -156,6 +157,10 @@ def button_handler(self, event):
             logger.info("launching Pi Noon challenge")
             new_challenge = PiNoon(timeout=self.timeout, screen=self.screen, joystick=self.joystick)
             return new_challenge
+        elif event.label == "Calibrate":
```

Comment:
We coiuld probably neaten this up a fair bit by defining a  label_name: class map, and then using that to initiate the new challenge. It doesn't need to be done for this PR though.